.buildkite/pipeline.yml

```diff
@@ -45,6 +43,18 @@ steps:
    - nix-build tests/tezos-binaries.nix --no-out-link --arg path-to-binaries ./docker
    branches: "!master"
 
+ - label: deb source packages via docker
+   commands:
+   - nix run -f. pkgs.docker -c ./docker/docker-ubuntu-packages.sh source
+   artifact_paths:
+     - ./out/*
+ # - label: deb binary packages via docker
+ #   commands:
+ #   - nix run -f. pkgs.docker -c ./docker/docker-ubuntu-packages.sh binary
+ #   artifact_paths:
+ #     - ./out/*
```

Comment:
AFAIU we still want to require it to pass before a PR is merged, but we will trigger it manually because it's an expensive action, right? And you will uncomment it once you setup BK this way.

Reply:
Yes, that's my plan, I've commented it for now in order to not slow down the CI